misc/atrium-specs/src/commonMain/kotlin/ch/tutteli/atrium/specs/integration/AbstractIterableToContainInAnyOrderAtLeast1EntriesExpectationsTest.kt

```diff
@@ -0,0 +1,225 @@
+        toContainInAnyOrderNullableEntriesSpec
+    ) { toContainInAnyOrderEntriesFun ->
+
+        it("$toBeLessThanFun(1.1) throws AssertionError") {
+            expect {
+                expect(emptyIterable()).toContainInAnyOrderEntriesFun({ toBeLessThan(1.1) }, arrayOf())
+            }.toThrow<AssertionError> {
+                message {
+                    toContain.exactly(1).values(
+                        "$rootBulletPoint$toContainInAnyOrder: $separator",
+                        "$anElementWhichNeedsDescr: $separator",
+                        "$toBeLessThanDescr: 1.1",
+                        noSuchElementDescr
+                    )
+                }
+            }
+        }
+        it("$toBeLessThanFun(1.1) and $toBeGreaterThanFun(2.1) throws AssertionError") {
+            expect {
+                expect(emptyIterable()).toContainInAnyOrderEntriesFun(
+                    { toBeLessThan(1.1) }, arrayOf({ toBeGreaterThan(2.1) })
+                )
+            }.toThrow<AssertionError> {
+                message {
+                    toContain.exactly(2).values(
+                        "$anElementWhichNeedsDescr: $separator",
+                        noSuchElementDescr
+                    )
+                    toContain.exactly(1).values(
+                        "$rootBulletPoint$toContainInAnyOrder: $separator",
+                        "$toBeLessThanDescr: 1.1",
+                        "$toBeGreaterThanDescr: 2.1"
+                    )
+                }
+            }
+        }
+    }
+
+    @TestFactory
+    fun one_to_seven() = testFactoryNonNullable(
+        toContainInAnyOrderEntriesSpec,
+        toContainInAnyOrderNullableEntriesSpec
+    ) { toContainInAnyOrderEntriesFun ->
+        describe("search for entry which $toBeGreaterThanFun(1.1) and $toBeLessThanFun(2.1)") {
+            it("throws AssertionError containing both assumptions in one assertion") {
+                expect {
+                    expect(oneToSeven()).toContainInAnyOrderEntriesFun(
+                        { toBeGreaterThan(1.1).toBeLessThan(2.1) }, arrayOf()
+                    )
+                }.toThrow<AssertionError> {
+                    message {
+                        toContain.exactly(1).values(
+                            "$rootBulletPoint$toContainInAnyOrder: $separator",
+                            "$anElementWhichNeedsDescr: $separator",
+                            "$toBeGreaterThanDescr: 1.1",
+                            "$toBeLessThanDescr: 2.1",
+                            noSuchElementDescr
+                        )
+                    }
+                }
+            }
+        }
+        describe("search for entry which $toBeGreaterThanFun(1.1) and $toBeLessThanFun(2.2)") {
+            it("does not throw an exception") {
+                expect(oneToSeven())
+                    .toContainInAnyOrderEntriesFun({ toBeGreaterThan(1.1) }, arrayOf({ toBeLessThan(2.2) }))
```

Comment:
> search for entry
You search for two entries here
```suggestion
                    .toContainInAnyOrderEntriesFun({ toBeGreaterThan(1.1) .toBeLessThan(2.2) }, emptyArray())
```

Reply:
@robstoll - Done